scripts/jest/test-framework-setup.js

```diff
@@ -3,6 +3,12 @@
 // We want to globally mock this but jest doesn't let us do that by default
 // for a file that already exists. So we have to explicitly mock it.
 jest.mock('ReactDOM');
+jest.mock('ReactDOMFeatureFlags', () => {
+  const flags = require.requireActual('ReactDOMFeatureFlags');
+  return Object.assign({}, flags, {
+    useFiber: !!process.env.REACT_DOM_JEST_USE_FIBER,
+  });
+});
```

Comment:
Yeah but it requires changing ReactDOMFeatureFlags.js which feels safe on Travis and not as safe if you might have local changes.